samples/corda/corda-simple-application/clients/src/main/resources/config/credentials/Corda_Network/access-control.json

```diff
@@ -5,5 +5,17 @@
     "principalType": "certificate",
     "resource": "localhost:30006#com.cordaSimpleApplication.flow.GetStateByKey:*",
     "read": true
+  },
+  {
+    "principal": "-----BEGIN CERTIFICATE-----\nMIIByzCCAWegAwIBAgIQaqyrhCxlulMXQ2ERwrN5FjAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcwLzELMAkGA1UEBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwG\nUGFydHlBMB4XDTIxMTAyMDAwMDAwMFoXDTI3MDUyMDAwMDAwMFowLzELMAkGA1UE\nBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwGUGFydHlBMCowBQYDK2Vw\nAyEAMaHQI58Jjpugv6uIZ1qej2YDAkYOd+8IngOkp1AXioCjgYkwgYYwHQYDVR0O\nBBYEFPI4LMY1d+hDVGJTXnDJuQviBQ59MA8GA1UdEwEB/wQFMAMBAf8wCwYDVR0P\nBAQDAgKEMBMGA1UdJQQMMAoGCCsGAQUFBwMCMB8GA1UdIwQYMBaAFM1cH1AsPx3G\nP0cFiOfCyk/ezevUMBEGCisGAQQBg4piAQEEAwIBBjAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcDSAAwRQIgcKAtTPzmuGtwGHTx4Gq07K0R96lGFgQhFCycFNgxcj4CIQD2\nE1rZxLU2pKar/MC86c+LNF1F55ehf9egcDUzJO8Bjg\u003d\u003d\n-----END CERTIFICATE-----",
+    "principalType": "certificate",
```

Comment:
Can we replace at least one of these policies with `principalType` as `ca` and `principal` as the MSP ID (for Fabric) or Node ID (for Corda)? That will allow us to continuously test the different policy verification paths too.

Reply:
Want to address this in my next PR that covers the test cases around asset transfer.